Bigtable/src/Connection/LongRunningConnection.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Connection;
+
+use Google\Cloud\Core\LongRunning\LongRunningConnectionInterface;
+
+/**
+ * Represents a connection to the Long Running Operations portion of the
+ * Cloud Bigtable API.
+ */
+class LongRunningConnection implements LongRunningConnectionInterface
+{
+    /**
+     * @var ConnectionInterface
+     */
+    private $connection;
+
+    /**
+     * @param ConnectionInterface $connection
+     */
+    public function __construct(ConnectionInterface $connection)
```

Comment:
Currently, the only ConnectionInterface in Bigtable is Grpc that you've implemented before.
But that doesn't include any of the below operations (e.g., getOperation, cancelOperation), right?
Shouldn't we submit the PR for adding those methods first or adding them here too?

Reply:
This still holds. Please add those methods in the ConnectionInterface too.